.travis.yml

```diff
@@ -1,32 +1,43 @@
-# Test build the MoveIt tutorials. Author: Dave Coleman
-sudo: required
-dist: xenial
-language: ruby
-rvm:
-  - 2.7
-python:
-  - "2.7"
+# This config file for Travis CI utilizes https://github.com/ros-planning/moveit_ci/ package.
+os: linux
+dist: bionic  # distro used by Travis, moveit_ci uses the docker image's distro
+services:
+  - docker
+language: cpp
+cache: ccache
+compiler: gcc
+
+notifications:
+  email: true
 
 env:
   global:
-    - SCRIPT=.moveit_ci/travis.sh
-    - ROS_DISTRO=noetic
-    - ROS_REPO=ros-testing
+    - MOVEIT_CI_TRAVIS_TIMEOUT=85  # Travis grants us 90 min, but we add a safety margin of 5 min
+    - ROS_DISTRO=foxy
+    - ROS_REPO=ros
+    - UPSTREAM_WORKSPACE=moveit2_tutorials.repos
     - CXXFLAGS="-Wall -Wextra -Wwrite-strings -Wunreachable-code -Wpointer-arith -Wredundant-decls"
     - WARNINGS_OK=false
-    - UPSTREAM_WORKSPACE="https://github.com/ros-planning/moveit_visual_tools
-                          https://github.com/ros-planning/panda_moveit_config"
-  matrix:
-    - SCRIPT=htmlproofer.sh
-    - TEST="clang-format catkin_lint"
-    - TEST="clang-tidy-fix"
```

Comment:
Any other tests you want to run? Should we run clang-tidy-fix?

Reply:
We need to find a way to keep htmlproofer in there

Reply:
That test  is included and passes. See lines 25 and 26 of the updated travis file. Also double check my message below on needing to update the script.